samples/server/petstore/rust-server/src/models.rs

```diff
@@ -139,38 +139,66 @@ impl ApiResponse {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct ArrayOfArrayOfNumberOnly {
-    #[serde(rename = "ArrayArrayNumber")]
-    #[serde(skip_serializing_if="Option::is_none")]
-    pub array_array_number: Option<Vec<Vec<f64>>>,
+#[derive(Debug, Clone, PartialEq, PartialOrd, Serialize, Deserialize)]
 
+pub struct ArrayOfArrayOfNumberOnly(object);
```

Comment:
Hm. Have been looking into the missing objects with a single property. I've added some logs that catch them as they pass through `postProcessModels` (printing out the datatype):

```
[main] WARN  o.o.c.languages.RustServerCodegen - Found a disappearing model: OuterNumber(OuterNumber), number
[main] WARN  o.o.c.languages.RustServerCodegen - Found a disappearing model: ArrayOfNumberOnly(ArrayOfNumberOnly), object
[main] WARN  o.o.c.languages.RustServerCodegen - Found a disappearing model: EnumClass(EnumClass), string
[main] WARN  o.o.c.languages.RustServerCodegen - Found a disappearing model: List(List), object
[main] WARN  o.o.c.languages.RustServerCodegen - Found a disappearing model: NumberOnly(NumberOnly), object
```

It looks as if those with datatype `object` turn up in `models.rs` with the datatype `object`, whilst those with other datatypes just disappear...